.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release Model Card Toolkit package to PyPI and TestPyPI
+
+on:
+  push:
+    paths:
+      - 'model_card_toolkit/**'
+      - 'setup.py'
+    branches:
+      - master
+      - r*
+  release:
+    types: [published]
+    tags:
+      - v*
+
+jobs:
+ build-and-publish:
+    name: Build Model Card Toolkit package and release to PyPI and Test PyPI
+    runs-on: ubuntu-latest
+    container:
+      image: gcr.io/tfx-oss-public/tfx_base:py37-20200729
```

Comment:
@codesue This is very nit, so apologies. Do we need to the `tfx-base` image here?

Reply:
This image was convenient because it has the right versions of the tools need to build the wheel, e.g. bazel. It's [used to build tfx wheels](https://github.com/tensorflow/tfx/blob/master/tfx/tools/docker/Dockerfile), which have similar build steps.

Reply:
Ah, perfect! Thank you for the explanation @codesue!